Sources/LanguageServerProtocol/SupportTypes/ClientCapabilities.swift

```diff
@@ -438,6 +448,8 @@ public struct TextDocumentClientCapabilities: Hashable, Codable {
 
   public var foldingRange: FoldingRange? = nil
 
+  public var callHierarchy: CallHierarchy? = nil
```

Comment:
This could use `DynamicRegistrationCapability` instead of adding a specific type for it.

Reply:
Done